Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -53,13 +59,12 @@ class PayPalDemoViewController: FeatureBaseViewController {
     // MARK: - FeatureBaseViewController Override
 
     /// Enable the PayPal Button once we have created an order
-    func createOrder(completion: @escaping (String?) -> Void = { _ in }) {
-        baseViewModel.createOrder(amount: amountTextField.text) { orderID in
-            if orderID != nil {
-                self.payPalButton.isEnabled = true
-            } else {
-                print("There was an error")
-            }
+    func createOrder() async {
+        let orderID = await baseViewModel.createOrder(amount: amountTextField.text)
+        if orderID != nil {
+            self.payPalButton.isEnabled = true
+        } else {
+            print("There was an error")
```

Comment:
Can we update this to be how a merchant would access the actual error and display the specific error message on the screen?

Reply:
Scratch that - I see we do this in the View Model.